supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -191,7 +191,7 @@ class JesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
   }
 
   private[jes] def generateJesOutputs(jobDescriptor: BackendJobDescriptor): Set[JesFileOutput] = {
-    val wdlFileOutputs = call.task.findOutputFiles(jobDescriptor.fullyQualifiedInputs, NoFunctions) map relativeLocalizationPath
+    val wdlFileOutputs = call.task.findOutputFiles(jobDescriptor.fullyQualifiedInputs, PureStandardLibraryFunctions) map relativeLocalizationPath
```

Comment:
It's because otherwise we weren't able to get engine function evaluations showing up as file outputs. 
E.g. in my motivating situation
```
output {
  String x = read_string(basename(foo, ".bam") + ".bai")
}
```
With no functions, that'll fail to evaluate and the file won't be delocalized for us
With pure functions, things that don't interact externally can still be evaluated, but not the full suite of functions (e.g. we still can't do `read_string(read_string(foo))`)